StabilityMatrix/App.xaml.cs

```diff
@@ -247,7 +247,9 @@ private void App_OnStartup(object sender, StartupEventArgs e)
                 });
 
             // Add Refit clients
-            serviceCollection.AddRefitClient<ICivitApi>(defaultRefitSettings)
+            // @NuarkNoir: When passing config to this method we are loosing sizeKB from response, who knows why.
+            // @NuarkNoit: Another solution, anyone?
+            serviceCollection.AddRefitClient<ICivitApi>()
```

Comment:
Hi, first of all, thanks for the PR!

The issue here is that we set the JsonPropertyName wrong in the `CivitFile.cs` class. If you change that file as such:
```diff
- [JsonPropertyName("sizeKb")]
+ [JsonPropertyName("sizeKB")]
public double SizeKb { get; set; }
```

it should work and you can add the `defaultRefitSettings` back to this here.

Reply:
👌